lego/apps/articles/migrations/0012_migrate_authors.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.9 on 2023-03-16 13:13
+
+from django.conf import settings
+from django.db import migrations
+
+
+def merge_authors(apps, schema_editor):
+    Article = apps.get_model("articles", "Article")
+    for e in Article.objects.all():
+        if e.created_by != None and e.authors == None:
```

Comment:
Should't this be `e.authors != None`, or can you do `.add()` anyways?

Reply:
It seemed to run fine for me. From testing in shell, the field is None if there's no entries and adding still works.